launcher/launcher/main.go

```diff
@@ -130,25 +144,35 @@ func getExitCode(isHardened bool, restartPolicy spec.RestartPolicy, err error) i
 
 func startLauncher() error {
 	logger.Println("Launch Spec: ", launchSpec)
-	client, err := containerd.New(defaults.DefaultAddress)
+	containerdClient, err := containerd.New(defaults.DefaultAddress)
 	if err != nil {
 		return &launcher.RetryableError{Err: err}
 	}
-	defer client.Close()
+	defer containerdClient.Close()
 
 	tpm, err := tpm2.OpenTPM("/dev/tpmrm0")
 	if err != nil {
 		return &launcher.RetryableError{Err: err}
 	}
 	defer tpm.Close()
 
+	// check EK cert
+	gceAk, err := client.GceAttestationKeyECC(tpm)
+	if err != nil {
+		return &launcher.RetryableError{Err: err}
+	}
+	if len(gceAk.CertDERBytes()) == 0 {
+		return &launcher.RetryableError{Err: errors.New("can't find AkCert on this VM, remote attestation is likely to fail! (use \"gcloud compute instances get-shielded-identity\" to check if the AkCert (ekCert signingKey) is present on the VM, if not, reboot the VM)")}
```

Comment:
It will fail. Our message should be more actionable than check getShieldedInstanceIdentity since they can't do anything else after that.

"Failed to find AKCert on this VM: try starting a new VM or contacting support."

Reply:
Done